example/lib/list/list_more.dart

```diff
@@ -0,0 +1,71 @@
+// SOURCE
+import 'dart:collection';
+
+import 'package:smartstruct/smartstruct.dart';
+
+part 'list_more.mapper.g.dart';
+
+class Source {
```

Comment:
Good Example. Could you also add tests, for the newly added functionalities?
Like this list mapping, Mapping of generic types

Reply:
Thank you! I will add the tests and examples which should be done from the beginning.